kratos/python/add_variable_utils_to_python.cpp

```diff
@@ -24,6 +24,31 @@
 // Variable utilities
 #include "utilities/variable_utils.h"
 
+#ifndef ADD_KRATOS_FLAGGED_ENTITY_DIFFERENT_VARIABLES_DATA_COPY_METHODS
```

Comment:
I think it is quite hard to understand what is happening here, but I don't have a good alternative suggestion

I did sth similar [here](https://github.com/KratosMultiphysics/CoSimIO/blob/master/co_sim_io/python/info_to_python.hpp#L31-L35) (=> instead of defining multiple functions you would define multiple types)
Maybe this could work together with [perfect forwarding](https://eli.thegreenplace.net/2014/perfect-forwarding-and-universal-references-in-c)

not sure though, check it yourself please

if it doesn't work then please go ahead, no complaints :)

Cheers for the effort

Reply:
I will try to think of something :)

Reply:
I removed the macros :)